pages/search.tsx

```diff
@@ -207,7 +207,7 @@ const Search: NextPage = () => {
                         <Card elevation="raised">
                           <CardHeader>
                             <CardHeaderTitle>
-                              <Typography variant="h4" as="p">
+                              <Typography variant="h4">
```

Comment:
I think we are safe to assume that we will at some point have a h1 as a general title for the search here? 🤔  So in that case, the correct header level would be `h2`

That being said, EDS doesn't separate design and semantic level in a good way for the Typography component. For the envis project, we have wrapped the Typography component to get by some of the shortcomings. But I think you can do `<Typography variant="h4" as="h2" />` to make it look better but still use the correct level 😄